cli/src/main/java/com/box/l10n/mojito/cli/command/PhabricatorExtractionCheckCommand.java

```diff
@@ -100,8 +91,8 @@
     @Parameter(names = {"--input-directory", "-i"}, arity = 1, required = false, description = ExtractionDiffCommand.INPUT_DIRECTORY_DESCRIPTION)
     String inputDirectoryParam = ExtractionPaths.DEFAULT_OUTPUT_DIRECTORY;
 
-    @Parameter(names = {"--context-comment-reject-pattern", "-ccrp"}, arity = 1, required = false, description = "Regex pattern that is used to identify context or comment strings that should be rejected.")
-    String contextCommentRejectPattern = null;
+    @Parameter(names = {"--checker-options", "-co"}, variableArity = true, required = false, listConverter = CheckerOptionsConverter.class, description = "Map of options to be used in individual checkers. Options must be in the form 'name':'value'.")
+    List<Map<String, String>> checkerOptions = new ArrayList<>();
```

Comment:
Given that we have a list option:value pairs - would a simple Map<String, String> work here to contain all of the values - or do we need multiple separate maps?
E.g.: could CheckerOptionsConverter return a MapEntry instead of a whole Map with only one value?
There might not be a cleaner to do that - just checking. (I saw someone else enquiring about similar functionality at some point: https://github.com/cbeust/jcommander/issues/218 )

Reply:
I think I should be able to clean it up to a custom map entry object and then map the stream of those objects into a Map (I need them in a map so that I can use the putAll method of ImmutableMap)